test/storage/Bridge

```diff
@@ -2,7 +2,7 @@
 ╭----------------------------------+------------------------------------------------+------+--------+-------+------------------------------╮
 | Name                             | Type                                           | Slot | Offset | Bytes | Contract                     |
 +==========================================================================================================================================+
-| _initialized                     | bool                                           | 0    | 0      | 1     | src/bridge/Bridge.sol:Bridge |
+| _initialized                     | uint8                                          | 0    | 0      | 1     | src/bridge/Bridge.sol:Bridge |
```

Comment:
In the release notes we mention a guide for dealing with storage changes but we should also include a link to that guide.

Reply:
still working on it